.github/workflows/deploy-alpha.yml

```diff
@@ -12,6 +12,7 @@ on:
 jobs:
   deployment:
     runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
```

Comment:
You should use needs instead.
CF a quick look at the doc : https://docs.github.com/en/actions/using-workflows/advanced-workflow-features#creating-dependent-jobs 😉 

Maybe we could redefine each step of the CI using this.
I don't know 🤷‍♂️

Reply:
needs is for jobs in the same file :/